src/java/htsjdk/samtools/Defaults.java

```diff
@@ -19,9 +19,18 @@
     /** Should MD5 files be created when writing out SAM and BAM files?  Default = false. */
     public static final boolean CREATE_MD5;
 
-    /** Should asynchronous I/O be used when writing out SAM and BAM files (one thread per file).  Default = false. */
+    /** Should asynchronous I/O be used when writing out SAM, BAM, FASTQ files (one thread per file).  Default = false. */
+    @Deprecated
```

Comment:
Also, you should mention explicitly in the docs that this option takes precedence over the other two.